media/js/app/panel.js

```diff
@@ -57,23 +68,6 @@
             jQuery(self.el).css('height', visible + 'px');
         };
 
```

Comment:
yeah.. I had overlooked one use of `panel.loadTemplates()` in that file. After adding back that function, it looks like the tests are getting a little farther.